cli/tests/integration/init_tests.rs

```diff
@@ -31,65 +25,52 @@ fn init_subcommand_without_dir() {
 
   assert!(cwd.join("deno.jsonc").exists());
 
-  let mut deno_cmd = util::deno_cmd_with_deno_dir(&deno_dir);
-  let output = deno_cmd
-    .current_dir(cwd)
+  let output = context
+    .new_command()
     .env("NO_COLOR", "1")
-    .arg("run")
-    .arg("main.ts")
-    .stdout(Stdio::piped())
-    .spawn()
-    .unwrap()
-    .wait_with_output()
-    .unwrap();
-  assert!(output.status.success());
-  assert_eq!(output.stdout, b"Add 2 + 3 = 5\n");
-
-  let mut deno_cmd = util::deno_cmd_with_deno_dir(&deno_dir);
-  let output = deno_cmd
-    .current_dir(cwd)
+    .args("run main.ts")
+    .split_output()
+    .run();
+
+  output.assert_exit_code(0);
+  assert_eq!(output.stdout().as_bytes(), b"Add 2 + 3 = 5\n");
+
+  let output = context
+    .new_command()
     .env("NO_COLOR", "1")
-    .arg("test")
-    .stdout(Stdio::piped())
-    .spawn()
-    .unwrap()
-    .wait_with_output()
-    .unwrap();
-  assert!(output.status.success());
-  let stdout = String::from_utf8(output.stdout).unwrap();
-  assert_contains!(stdout, "1 passed");
-
-  let mut deno_cmd = util::deno_cmd_with_deno_dir(&deno_dir);
-  let output = deno_cmd
-    .current_dir(cwd)
+    .args("test")
+    .split_output()
+    .run();
+
+  output.assert_exit_code(0);
+  assert_contains!(output.stdout(), "1 passed");
+  output.skip_output_check();
```

Comment:
@dsherret It feels like I should not have to call `skip_output_check` after calling `output.stdout()` or `stderr()` but I get errors unless I add it in atm

Reply:
Yeah, you have to assert both stdout and stderr when non-empty. I can improve the message later.

Reply:
Do you think I should replace them? With e.g. `assert!(output.stderr().contains("Check"));` or `assert!(!output.stderr().is_empty());`?

Reply:
This is fine.